spec/linter-eslint-spec.js

```diff
@@ -162,13 +162,21 @@ describe('The eslint provider for Linter', () => {
     beforeEach(() => {
       atom.config.set('linter-eslint.disableEslintIgnore', false)
     })
-    it('will not give warnings for the file', () => {
+    it('will not give warnings when linting the file', () => {
       waitsForPromise(() =>
         atom.workspace.open(ignoredPath).then(editor =>
           lint(editor).then(messages => expect(messages.length).toBe(0))
         )
       )
     })
+
+    it('will not give warnings when autofixing the file', () => {
+      waitsForPromise(() =>
+        atom.workspace.open(ignoredPath).then(editor =>
+          fix(editor).then(result => expect(result).toBe('Linter-ESLint: Fix complete.'))
```

Comment:
I think you mean we used to show the 'file ignored' messages when linting. And yeah, that's noisy. But when a user manually runs the fix command, we should show a meaningful message. I'll push a PR later if I can get around to it.